arch/csr/hcontext.yaml

```diff
@@ -0,0 +1,49 @@
+# yaml-language-server: $schema=../../schemas/csr_schema.json
+
+$schema: "csr_schema.json#"
+kind: csr
+name: hcontext
+long_name: Hypervisor-mode context register
+address: 0x6A8
+priv_mode: S
+length: XLEN
+description: |
+  This optional register may be implemented only if the H extension is implemented.
+  If it is implemented, mcontext must also be implemented.
+
+  This register is only accessible in HS-Mode, M-mode and Debug Mode. If Smstateen is implemented,
+  then accessibility of in HS-Mode is controlled by mstateen0[57].
+
+  This register is an alias of the mcontext register, providing access to the hcontext field from HS-Mode.
```

Comment:
backtick mcontext.
uppercase HCONTEXT to match the representation in "alias", below.